engine/engine/src/wscript

```diff
@@ -155,7 +160,7 @@ def build(bld):
 
     obj = bld(
         features = 'c cxx cprogram apk web extract_symbols',
-        use = 'WEBVIEWEXT PROFILE_NULL PROFILEREXT_NULL FACEBOOKEXT IAPEXT PUSHEXT IACEXT GAMEOBJECT DDF LIVEUPDATE RESOURCE GAMESYS PHYSICS RECORD RENDER PLATFORM_SOCKET SCRIPT LUA EXTENSION HID INPUT PARTICLE RIG DLIB GUI CARES CRASH X engine_release engine_service_null'.split() + graphics_lib + sound_lib + additional_libs,
+        use = 'WEBVIEWEXT PROFILE_NULL PROFILEREXT_NULL FACEBOOKEXT IAPEXT PUSHEXT IACEXT GAMEOBJECT DDF LIVEUPDATE RESOURCE GAMESYS PHYSICS RECORD RENDER PLATFORM_SOCKET SCRIPT LUA EXTENSION HID PLATFORM_HID INPUT PARTICLE RIG DLIB GUI CRASH X engine_release engine_service_null'.split() + graphics_lib + sound_lib + additional_libs,
```

Comment:
I think I need to take another pass later on to consolidate how we specify the libraries.
E.g I see no reason why we should have both a "HID", and "PLATFORM_HID", when we can merge them.